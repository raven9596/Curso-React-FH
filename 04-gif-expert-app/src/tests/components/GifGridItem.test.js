import React from 'react';
import {shallow} from 'enzyme';
import { GifGridItem } from '../../components/GifGridItem';


describe('Prueba en <GifGridItem />', () => {

  test('debe de mostrar el componente correctamente ', () => {

    const title = 'Dragon Ball Z';
    const url = 'https://localhost/algo.jpg';
    const wrapper = shallow(<GifGridItem title={ title } url={ url } />);

    expect( wrapper ).toMatchSnapshot();
    
  });
  
});