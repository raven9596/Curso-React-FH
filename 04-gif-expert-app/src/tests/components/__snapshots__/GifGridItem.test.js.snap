// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <GifGridItem /> debe de mostrar el componente correctamente  1`] = `
<div
  className="card animate__animated animate__fadeIn animate__delay-2s"
>
  <img
    alt="Dragon Ball Z"
    src="https://localhost/algo.jpg"
  />
  <p>
    Dragon Ball Z
  </p>
</div>
`;
